source/_static/badges/ent-cloud-dedicated.rst

```diff
@@ -0,0 +1,18 @@
+:orphan:
+:nosearch:
+
+.. raw:: html
+
+  <div class="mm-badge">
+
+|plans-img| Available only on `Enterprise <https://mattermost.com/pricing/>`__ plans
```

Comment:
@spirosoik I am not sure if here we want the pricing page or directly the Cloud Enterprise Page.

Reply:
@spirosoik Also in this page we might want somowhere the talk to an expert link or button

Reply:
This needs to redirect you here I believe https://mattermost.com/contact-sales/

Reply:
or here https://mattermost.com/enterprise/cloud/ I cannot decide which one fits best